docs/docs/agents/memory.md

```diff
@@ -149,6 +149,75 @@ agent = create_react_agent(
 
 To learn more about using `pre_model_hook` for managing message history, see this [how-to guide](../how-tos/create-react-agent-manage-message-history.ipynb)
 
+### Read in tools { #read-short-term }
+
+LangGraph allows agent to access its short-term memory (state) inside the tools. See the [Context](./context.md#__tabbed_2_2) guide for more information.
+
+### Write from tools { #write-short-term }
+
+To modify the agent's short-term memory (state) during execution, you can return state updates directly from the tools. This is useful for persisting intermediate results or making information accessible to subsequent tools or prompts.
+
+```python
+from typing import Annotated
+from langchain_core.tools import InjectedToolCallId
+from langchain_core.runnables import RunnableConfig
+from langchain_core.messages import ToolMessage
+from langgraph.prebuilt import InjectedState, create_react_agent
+from langgraph.prebuilt.chat_agent_executor import AgentState
+from langgraph.types import Command
+
+class CustomState(AgentState):
+    # highlight-next-line
```

Comment:
We likely have too many highlights to make the highlights useful.

Let's focus just on `Command(update=...)` and remove all other highlights

Reply:
yea, was also concerned about this one